api/api.yaml

```diff
@@ -199,3 +221,8 @@ definitions:
         description: >
           If this search result represents a facet, facet_value is the empty string.
           If this search result represents a facet and selected value, facet_value is the selected value.
+      is_time_series:
```

Comment:
Why is this needed? (I haven't read the code yet)

Reply:
It's used to determine when to add the "at {time series unit} {time}" clause to the search option. Should this go in the description?

Reply:
Ah ok. This doesn't need to go in description. But please make boolean instead of integer.

(The logic in Search.js is getting complicated. In general I prefer to have complicated logic in API server and not UI server. So in retrospect it might be better to have API server just return es_field_name and the html of what to render. Probably not worth changing at this point.)